{{cookiecutter.github_repository}}/pyproject.toml

```diff
@@ -0,0 +1,147 @@
+[tool.poetry]
+name = "{{cookiecutter.github_repository}}"
+version = "{{cookiecutter.version}}"
+description = "{{cookiecutter.project_description}}"
+authors = ["{{cookiecutter.default_from_email}}"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+Django = "^3.2.10"
```

Comment:
Caret notation can be dangerous here. As django doesn't following semver, this line means poetry can update django until <4.0. For current value of `^3.2.10` this works, but if we have a value of `^4.0` then poetry may update to `4.1` or `4.2` which we know are major upgrades. I suggest we use _tilde requirements_ for django and other deps where they don't follow semver. More here https://python-poetry.org/docs/dependency-specification/

Reply:
Fixed. Thanks!